book/website/collaboration/github-novice.md

```diff
@@ -17,7 +17,7 @@ However, it has evolved into a collaborative platform for developing documents a
 Many of the new tools developed for reproducible research can seem daunting to researchers who have little or no experience with computational tools.
 However, if researchers have used online tools such as Google workspace and Microsoft office, with a small amount of instruction they should be able to learn to use GitHub online platform fairly easily.
 
-In this chapter, we have describe the {ref}`motivation of using GitHub<cl-github-novice-motivation>`, explained the {ref}`important first steps<cl-github-novice-firststeps>`, discussed some {ref}`useful features<cl-github-novice-features>` and listed resources to learn {ref}`advanced GitHub features<cl-github-novice-advanced>`.
+In this chapter, we have described the {ref}`motivation of using GitHub<cl-github-novice-motivation>`, explained the {ref}`important first steps<cl-github-novice-firststeps>`, discussed some {ref}`useful features<cl-github-novice-features>` and listed resources to learn {ref}`advanced GitHub features<cl-github-novice-advanced>`.
```

Comment:
```suggestion
In this chapter, we have describe the {ref}`motivation of using GitHub<cl-github-novice-motivation>`, explained the {ref}`important first steps<cl-github-novice-firststeps>`, discussed some {ref}`useful features<cl-github-novice-features>` and listed resources to learn {ref}`advanced GitHub features<cl-github-novice-advanced>`.
```

This has since been updated [elsewhere](https://github.com/alan-turing-institute/the-turing-way/commit/e584dfe1a444a5c090fa2644d64ce2218f6af668), so I'm reverting it with the goal of working around the git conflict.